### ISSUE TITLE
Bike Connectivity to Bluetooth

### ISSUE THREAD

**Issue by VincentSaelzler** [ericsakmar/connected-bike#2]
Where in the code does the Bluetooth connection with the bike happen?

This is the closest thing I can find:

https://github.com/ericsakmar/connected-bike/blob/76a5957fecc8f5b1d44f0f7e5c4c8fbf56527fcd/src/services/bikeDataService.js#L9

I am trying to implement something similar! I would love to utilize what you have already written if possible.

Thanks!

**Comment by ericsakmar** [comment]
Yes, that's the spot. The browser will present a list of devices that match the options, and you can select one from there.

**Comment by VincentSaelzler** [comment]
Thanks for clarifying!

I was trying to use Firefox. According to this documentation (ironically from Mozilla) it appears that Web Bluetooth is **not** supported in Firefox.

https://developer.mozilla.org/en-US/docs/Web/API/Bluetooth/requestDevice#browser_compatibility

I will give it another try using Chrome/Chromium.
